binding.gyp

```diff
@@ -28,7 +28,7 @@
       'target_name': '<(module_name)',
       'product_dir': '<(module_path)',
       'dependencies': [ 'action_before_build' ],
-      'sources': [ './src/hello_world.cpp' ],
+      'sources': [ './src/module.cpp', './src/hello.cpp' ],
```

Comment:
We could also do a "glob" here. Something like: 

```
"<!@(find ./src -name '*.cpp')"
```

That would avoid needing to add files explicitly. If you think this makes sense @GretaCB - I'd recommend adding this.

Caveat however: is that `find` won't work on windows. I don't think we need to support windows, but if we ever did we could use the cross-platform approach using python at https://github.com/mapnik/mapnik-gyp/blob/d8b21e7698334fc9e574d330ed8bd9db061cb8e1/mapnik.gyp#L122